_projects/coala_language_server.md

```diff
@@ -0,0 +1,63 @@
+---
+collaborating_projects:
+  - coala
+desc: "Offer coala languager server."
+difficulty: medium
+initiatives:
+  - GSoC
+  - RGSoC
+issues:
+  - "https://github.com/coala/coala-vs-code/issues/3"
+  - "https://github.com/coala/coala-vs-code/issues/4"
+markdown: coala_language_server.md
+mentors:
+  - gaocegege
+name: "coala Language Server"
+requirements:
+  - "The applicant should have at least one patch accepted to any of the coala."
```

Comment:
Any of the coala? Maybe add "Repository"

Reply:
OK